src/liblx/xml/test/TestXMLInputStream.c

```diff
@@ -102,11 +102,11 @@ START_TEST (test_XMLInputStream_next_peek)
   
   fail_unless(stream != NULL);
   
-  fail_unless(strcmp(XMLToken_getName(next0), "sbml") == 0);
+  fail_unless(strcmp(XMLToken_getName(next0), "sbml" /*"lx"*/) == 0);
```

Comment:
not sure what's happening here? Is this why lx is in comments?

Reply:
I think this is because of the gradual transition from "sbml" to "liblx". This particular one might be explained by the other PR (which adds the other twoXML libraries in as build options).